modules/oh_sanitizer/field.ebnf

```diff
@@ -5,20 +5,36 @@
 // - For English, French, Italian and Spanish
 //   Note: Make sure to avoid collisions between languages for the
 //         abbreviations.
-WDAY_MO : "Monday"i | "Lundi"i | "Lunedi"i | "Lunes"i
-        | "Mon"i | "Lun"i | "Mo"i | "Lu"i
-WDAY_TU : "Tuesday"i | "Mardi"i | "Martedì"i | "Martes"i
-        | "Tue"i | "Tu"i | "Ma"i // Note: "Mar" would collide with March
+WDAY_MO : "Monday"i | "Lundi"i | "Lunedi"i | "Lunes"i | "Понедельник"i
+        | "Mon"i | "Lun"i | "Пон"i 
+        | "Mo"i | "Lu"i | ("По"i | "Пн"i | "Пк"i)
```

Comment:
Please can you remove all parenthesis.

Reply:
Updated in new code.  I was not sure if you also wanted those changed in older code (see Wednesday, Saturday lines).